core/optic/js/src/main/scala/com/useoptic/QueriesJsFacade.scala

```diff
@@ -22,12 +21,13 @@ import scala.scalajs.js.annotation.{JSExport, JSExportAll, JSExportTopLevel}
 @JSExport
 @JSExportAll
 object OASProjectionHelper {
-  def fromEventString(eventString: String): js.Any = {
+  //@TODO: pass in apiName
```

Comment:
If I understand correctly, we are no longer using this part of the event stream and instead we are using the value from the optic.yml correct? I'm taking this opportunity to get rid of dead code.

Reply:
Yes sir, API name should come from `optic.yml`, it's not currently used anywhere in the UX so safe to make changes here